pkgs/development/python-modules/picos/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, numpy
+, cvxopt
+}:
+
+buildPythonPackage rec {
+  pname = "picos";
+  version = "1.2.0";
+
+  src = fetchFromGitLab {
+    owner = "picos-api";
+    repo = "picos";
+    rev = "b9c3105b31166c307296dee6e0c26cf6a0ab8723";
+    sha256 = "018xhc7cb2crkk27lhl63c7h77w5wa37fg41i7nqr4xclr43cs9z";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    cvxopt
+  ];
+
+  doCheck = true;
+  checkPhase = ''
+    python ${src}/test.py
+  '';
+  
+  meta = with lib; {
+    description = "User friendly Python API to several conic and integer programming solvers, very much like YALMIP or CVX under MATLAB";
```

```suggestion
    description = "A Python interface to conic optimization solvers";
```

shorter description on gitlab